app/config.py

```diff
@@ -106,6 +106,7 @@
     )
 DATABASE_URL = DATABASE_URL.replace("postgresql://", "postgresql+psycopg://")
 DB_ECHO = True if CONFIG["database"]["echo"] == "yes" else False
+DATABASE_TYPE = "postgresql" if DATABASE_URL.startswith("postgresql") else "mysql"
```

Comment:
`DATABASE_TYPE` is added to discriminate which DB type is used before DB engine is generated.